frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerEllipsisMenu/containerEllipsisMenu.component.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { useParams } from 'react-router';
+import { IContainer } from '@/v5/store/containers/containers.types';
+import { formatMessage } from '@/v5/services/intl';
+import { ContainersActionsDispatchers } from '@/v5/services/actionsDispatchers/containersActions.dispatchers';
+import { DialogsActions } from '@/v5/store/dialogs/dialogs.redux';
+import { useDispatch } from 'react-redux';
+import { EllipsisMenu } from '@controls/ellipsisMenu/ellipsisMenu.component';
+import { EllipsisMenuItemLink } from '@controls/ellipsisMenu/ellipsisMenuItem/ellipsisMenutItem.component';
+
+type ContainerEllipsisMenuProps = {
+	selected: boolean,
+	container: IContainer,
+	onSelectOrToggleItem: (id: string) => void,
+	openShareModal: () => void,
+};
+
+export const ContainerEllipsisMenu = ({
+	selected,
+	container,
+	onSelectOrToggleItem,
+	openShareModal,
+}: ContainerEllipsisMenuProps) => {
+	const { teamspace, project } = useParams() as { teamspace: string, project: string };
+	const dispatch = useDispatch();
+
+	return (
+		<EllipsisMenu>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.loadContainer',
+					defaultMessage: 'Load Container in 3D Viewer',
+				})}
+				to=""
+				onClick={() => { }}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.loadContainer',
+					defaultMessage: 'Load Container in 3D Viewer',
+				})}
+				to={`/${container._id}`}
+				onClick={() => { }}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.uploadNewRevision',
+					defaultMessage: 'Upload new Revision',
+				})}
+				to=""
+				onClick={() => { }}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.viewIssues',
+					defaultMessage: 'View Issues',
+				})}
+				to=""
+				onClick={() => { }}
+
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.viewRisks',
+					defaultMessage: 'View Risks',
+				})}
+				to=""
+				onClick={() => { }}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage(selected
+					? { id: 'containers.ellipsisMenu.hideRevisions', defaultMessage: 'Hide Revisions' }
+					: { id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' })}
+				to=""
+				onClick={() => onSelectOrToggleItem(container._id)}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.editPermissions',
+					defaultMessage: 'Edit Permissions',
+				})}
+				to=""
+				onClick={() => { }}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.shareContainer',
+					defaultMessage: 'Share Container',
+				})}
+				to=""
+				onClick={openShareModal}
+			/>
+			<EllipsisMenuItemLink
+				title={formatMessage({
+					id: 'containers.ellipsisMenu.settings',
+					defaultMessage: 'Settings',
+				})}
+				to=""
+				onClick={() => { }}
```

Comment:
lets get rid of the empty 'tos' and 'onclick'

Reply:
done